plugins/generator-1.18.2/forge-1.18.2/templates/json/item_cmodel_blocking.json.ftl

```diff
@@ -0,0 +1,19 @@
+{
+  "parent": "${modid}:item/${registryname}",
+  "display": {
+    "thirdperson_righthand": {
+        "rotation": [ 45, 135, 0 ]

```

Comment:
Why didn't you subtract `rotation` values from each other here (blocking and normal)? Because it looks like it's what you did to `translation` parameters.

Reply:
that is what I did initially, but it ended up looking messed up. with the current rotations/translations it looks like a regular shield's blocking.